zossrc/makefile

```diff
@@ -0,0 +1,70 @@
+ sscom,\                                                               
+ nolongname,\                                                          
+ inline,\                                                              
+ genasm,\                                                              
+ inlrpt,\                                                              
+ csect,\                                                               
+ nose,\                                                                
+ lp64,\                                                                
+ list,\                                                                
+ warn64,\                                                              
+ optimize(2),\                                                         
+ list,\                                                                
+ showinc,\                                                             
+ showmacro,\                                                           
+ source,\                                                              
+ aggregate                                                             
+                                                                       
+MTL_FLAGS = -S -W "c,$(MTL_OPTS)"                  
+                                                                       
+MACLIBS = -ISYS1.MACLIB \                                              
+ -ICBC.SCCNSAM                                                         
+                                                                       
+MTL_HEADERS = -I/usr/include/metal                                     
+                                                                       
+DLL_CPP_FLAGS = -W "c,lp64,langlvl(extended),dll,XPLINK,exportall" \     
+ -qsearch=/sys/java64bt/v8r0m0/usr/lpp/java/J8.0_64/include \          
+ -qsource \                                                            
+ -c                                                                    
+                                                                       
+DLL_BND_FLAGS = -W "l,lp64,dll,dynam=DLL,XPLINK,map,list" \              
+ -qsource                                                              
+                                                                       
+ASM = as                                                               
+ASM_FLAGS = -mrent                                                     
+                                                                       
+libwtojni.so: wtojni.o wtoexec.o                                       
+	$(CXX) $(DLL_BND_FLAGS) -o $@ $¬ > $*.bind.lst                          
+	extattr +p $@                                                         
+                                                                       
+wtojni.o: wtojni.cpp                                                   
+	$(CXX) $(DLL_CPP_FLAGS) -qlist=$*.cpp.lst -o $@ $¬                      
```

Comment:
Does the `¬` work properly if you upload/download via ftp/zowe? I know we have a problem using ISPF and the `^` character.

Reply:
yeah - I messed that up - fixed in the latest push